callouts/python/README.md

```diff
@@ -360,3 +361,31 @@ docker build \
   -f ./extproc/example/jwt_auth/Dockerfile \
   -t service-callout-jwt-example-python .
 ```
+
+# Docker with TCP extension
```

Comment:
Not suggested it has to be added in this PR, but a bunch of the documentation up to this point in the file (e.g. the "Developing Callouts" section) is specific to the HTTP-level callout server, and doesn't address L4 callouts. How about tweaking the title of that section to make clear that it is specific to L7 callouts, and adding a placeholder for a similar section for L4 callouts?

Reply:
agreed, added more L7 marker, and will add more for the L4 at the end of this Readme.